Gem/AWS/MetricsCDK/aws_metrics/aws_metrics_constants.py

```diff
@@ -0,0 +1,140 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+# Constants for API Gateway Service API
+APIGATEWAY_STAGE = "api"
+
+
+# Constants for Kinesis Data Analytics application
+# The SQL code for the Kinesis analytics application.
+KINESIS_APPLICATION_CODE = "-- ** Continuous Filter **\n"\
```

Comment:
can we indent the nested clauses [like how its done](https://github.com/o3de/o3de/blob/development/Gems/AWSMetrics/cdk/aws_metrics/aws_metrics_constants.py#L23-L34) in the AWSMetrics gem? it really helps readability.

Reply:
Sure. Just made the change.